services/drone/drone-build.tester.js

```diff
@@ -0,0 +1,71 @@
+'use strict'
+
+const Joi = require('joi')
+const { isBuildStatus } = require('../build-status')
+const t = (module.exports = require('../tester').createServiceTester())
+const { mockDroneCreds, token, restore } = require('./drone-test-helpers')
+
+// Drone build (cloud)
```

Comment:
just a minor nit, but you can probably remove these comments. the test names you've used sufficiently describe the tests IMO

Reply:
Done!